omise/payment_methods/alipay.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+if (defined('_PS_MODULE_DIR_')) {
+    require_once _PS_MODULE_DIR_ . 'omise/payment_methods/_offsite.php';
+}
+
+class OmisePaymentMethod_Alipay extends OmiseOffsitePaymentMethod
```

Comment:
Understand that the `_` is on a purpose of automation, but I think we should strict with our project naming convention (it's not clearly set but I think the original approach is just simple camelCase without underscore).

Reply:
Nothing to do with automation. It is my convention to let us know that the file contains merely a parent/base/utility classes, not a payment method that will actually be used. The underscores make the look different and keep them at the head of a file list

Reply:
I may have misunderstood - are you referring to file names or class names? If class name then I'd like to keep them as is. It was a conscious choice for readability - `OmisePaymentMethod_XXX`

Reply:
Yup I was mentioning about its class name. (nothing to add more, just reply to your question)